micrometer-jetty12/build.gradle

```diff
@@ -11,17 +11,13 @@ dependencies {
     optionalApi libs.jetty12Server
     optionalApi libs.jetty12Client
 
-    testRuntimeOnly(libs.logbackLatest) {
-        version {
-            strictly libs.logbackLatest.get().version
-        }
-    }
+    testRuntimeOnly(libs.logbackLatest)
```

Comment:
A benefit of getting rid of build-wide dependency management is we don't have to fight it in cases like this where we want to use a different version in a project.